builder/internals.go

```diff
@@ -603,11 +603,10 @@ func (b *Builder) run(c *daemon.Container) error {
 
 	// Wait for it to finish
 	if ret, _ := c.WaitStop(-1 * time.Second); ret != 0 {
-		err := &jsonmessage.JSONError{
+		return &jsonmessage.JSONError{
```

Comment:
Ohhhh I got it now! However because that error is also a field in jsonmessage struct and it 's (once) checked against nil and with Value it does not work